docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,90 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
+
+The `notification` command provides a way for managing your [Meltano Cloud notifications](concepts#meltano-cloud-notifications).
+
+#### `list`
+
+List all notifications configured for your current default project or organization.
+
+```sh
+$ meltano cloud config notification list
+╭─────────────────────┬─────────┬──────────╮
+│ Recipient           │ Type    │ Status   │
+├─────────────────────┼─────────┼──────────┤
+│ https://melty       │ webhook │ active   │
+│ dragon@meltano.com  │ email   │ active   │
+╰─────────────────────┴─────────┴──────────╯
+```
+
+#### `set`
+
+Set a new notification or override an existing one.
+
+##### Case-Sensitivity in Values
+
+Take note that all the values in the commands and configuration settings are case-sensitive. This means job_created, Job_Created, and JOB_CREATED would be considered as different events. Please ensure to use the correct casing to avoid configuration issues or unexpected behaviors.
+
+
+Interactive mode
+```sh
+meltano cloud config notification set <type>
+```
+
+Non-interactive mode:
+```sh
+meltano cloud config notification set <type> --recipient <target_recipient> --filter <filter>
+````
+
+**`options`**
+
+`--recipient` (Required): The recipient to send the notification to.
+
+`--filter`: An optional filter you want to check against, you can supply multiple filters which will all be joined as an OR expression.
+Values within the filter are validated as an AND Expression.
+
+```sh
+meltano cloud config notification set webhook --recipient http://test+melty --filter '{"events": ["job_created", "job_succeeded"], "status": ["succeeded"]}' --filter '{ "status": ["failed"]}'
+```
+The above example sets a webhook notification to send to ﻿http://test+melty if the event is ﻿(job_created OR job_succeeded AND status = succeeded) OR status = failed.
+
+#### `update`
+
+Update an existing notification.
+
+```sh
+meltano cloud notification update webhook --recipient <old_url> --new <new_recipient> --status <status>
+```
+
+**`options`**
+
+
+`--recipient` (Required): The recipient of the notification to update.
+
+`--new`: The new recipient value
+
+`--status`: The new status of the notification (either 'active' or 'inactive')
+
+```sh
+meltano cloud config notification update --recipient http://test+melty --new http://test+better --status inactive
+```
+
+The above example updates the notification that was configured to send to `http://test+melty`. It changes the recipient to `http://test+better` and sets the status to inactive.
+
+#### `delete`
+Delete an existing notification
+
+```
+meltano cloud notification delete webhook --recipient <webhook_url>
```

Comment:
@leoroese what happens if I have multiple notifications going to the same recipient? When I run the delete does it delete them all?

Reply:
Currently we only support a single notification per recipient per (scope_and_id) combo, which means either 'organization', or 'project', or 'user'. If you were to set another one to the same recipient in the same scope, it overrides your previous notification to that recipient in that scope. If you then delete that notification it only deletes the single notification

Reply:
Let's add that as a comment in this section then. Good to document the limitations.